### ISSUE TITLE
Version 2 and z_index

### ISSUE THREAD

**Issue by feendrache** [Kiamo2/YATI#52]
**Describe the bug**
All Imports are working well, except that the custom attribute "z_index" is not imported into Godot anymore

**To Reproduce**
1. Open a Map in Tiled
2. Add custom attribute "z_index" other than 0 and save
3. reimport map in godot

**Expected behavior**
zIndex value should be used in tileMapLayer

**Comment by Kiamo2** [comment]
Hmm, TilemapLayer in 4.3 has no longer a specific property 'z_index' (while the Tilemap Layers in 4.2 had this property).
That's the reason it's not imported in 2.0
Probably the Canvas property 'z_index' should now take over?
Is that what you would expect as behavior?

**Comment by feendrache** [comment]
Yes, it would be the canvasItem z_index would be where it should go, that would use the "old behaviour" it had

**Comment by Kiamo2** [comment]
Ok, I'll fix that asap.

**Comment by feendrache** [comment]
you're the best

**Comment by Kiamo2** [comment]
(Hopefully) fixed in v2.0.1

**Comment by feendrache** [comment]
Yes, works now :)
